### PR TITLE
add intro paragraph and data source info to readme

### PR DESCRIPTION
**Pull request by elmiko**

### CODE REVIEW

**Comment by elmiko** [comment]
@willb ptal

**Comment by elmiko** [comment]
i just want to make sure the language on the data set stuff is ok

File: README.md [hunk 1]
```diff
@@ -1,2 +1,38 @@
-# streaming-lab
-Artifacts and resources to support the streaming and event processing labs for radanalytics.io
+# radanalytics.io streaming and event processing lab
+
+This repository contains artifacts and resources to support the streaming and
+event processing labs for radanalytics.io.
+
+## Source data
+
+The source data for this lab is imagined as a series of [Twitter](https://twitter.com)
```
Comment (willb) [inline README.md]: nit:  really the kind of data are "synthetic social media updates"

File: README.md [hunk 2]
```diff
@@ -1,2 +1,38 @@
...
+tweets. The text from these tweets will be used in conjunction with sentiment
+analysis to help demonstrate the use of [Apache Spark](https://spark.apache.org)
+and machine learning to investigate data. The data used for this lab is
+randomly generated using
+[Markov chains](https://en.wikipedia.org/wiki/Markov_chain), none of this data
```
Comment (willb) [inline README.md]: nit: comma splice

File: README.md [hunk 3]
```diff
@@ -1,2 +1,38 @@
...
+is from live Twitter accounts and it contains no personally indentifiable information.
+
+The format used for transmitting the Tweet data on the wire is defined by
+this [JSON Schema](http://json-schema.org) notation:
+
+```
+{
```
Comment (willb) [inline README.md]: actual schema looks good.  We could (for ease of use) also add a list of identified hashtags, although those are easy enough to parse out.

**Review by willb — APPROVED** [review]
LGTM.  Feedback is mostly superficial.

File: README.md (hunk 1 above)
Comment (elmiko) [reply README.md]: ok, changed the wording here a little bit. removed all the twitter/tweet references.

File: README.md (hunk 2 above)
Comment (elmiko) [reply README.md]: done

File: README.md (hunk 3 above)
Comment (elmiko) [reply README.md]: i'm good with adding the hashtags too, maybe just add that as a followup PR once you add it to the emitter?

**Comment by elmiko** [comment]
thanks @willb , updated with some changes

**Pull request merged**